### PR TITLE
feat(theme): updated default language properties

### PR DESCRIPTION
**Pull request by ashraf706**
English and french language properties updated.
All the templates have been updated to incorporate the changes.

Github issue: https://github.com/gravitee-io/issues/issues/8054

### CODE REVIEW

File: gravitee-am-gateway/gravitee-am-gateway-standalone/gravitee-am-gateway-standalone-distribution/src/main/resources/templates/i18n/messages_en.properties [hunk 1]
```diff
@@ -24,3 +31,128 @@ reset_password.email.header.title=Hi {0}, there was a request to reset your pass
...
+mfa.http=HTTP
+mfa.fido=FIDO2
+mfa.recovery.code=Recover Code
+
+mfa_enroll.title=Select a method
+mfa_enroll.description=Select the authentication method you want to pair with your account
+mfa_enroll.button.submit=Next
+mfa_enroll.button.skip=Skip for now
+mfa_enroll.button.back=Back to methods
+mfa_enroll.otp.description=Use and authenticator app to authenticate
```
Comment (farmborough) [inline gravitee-am-gateway/gravitee-am-gateway-standalone/gravitee-am-gateway-standalone-distribution/src/main/resources/templates/i18n/messages_en.properties:95]:
```suggestion
mfa_enroll.otp.description=Use an authenticator app to authenticate
```

File: gravitee-am-gateway/gravitee-am-gateway-standalone/gravitee-am-gateway-standalone-distribution/src/main/resources/templates/i18n/messages_en.properties [hunk 2]
```diff
@@ -24,3 +31,128 @@ reset_password.email.header.title=Hi {0}, there was a request to reset your pass
...
+mfa_enroll.otp=Scan the QR code with your authentication app (Google Authenticator or FreeOTP) and enter the code displayed in the next step
+mfa_enroll.sms.description=Receive a verification code to your phone to authenticate
+mfa_enroll.sms=We will send verification code to your phone number
```
Comment (farmborough) [inline gravitee-am-gateway/gravitee-am-gateway-standalone/gravitee-am-gateway-standalone-distribution/src/main/resources/templates/i18n/messages_en.properties:98]:
```suggestion
mfa_enroll.sms=We will send a verification code to your phone number
```

File: gravitee-am-gateway/gravitee-am-gateway-standalone/gravitee-am-gateway-standalone-distribution/src/main/resources/templates/i18n/messages_en.properties [hunk 3]
```diff
@@ -24,3 +31,128 @@ reset_password.email.header.title=Hi {0}, there was a request to reset your pass
...
+mfa_enroll.call=Invalid phone number
+mfa_enroll.email.description=Send email to your email address
```
Comment (farmborough) [inline gravitee-am-gateway/gravitee-am-gateway-standalone/gravitee-am-gateway-standalone-distribution/src/main/resources/templates/i18n/messages_en.properties:100]:
```suggestion
mfa_enroll.email.description=Send an email to your email address
```

File: gravitee-am-gateway/gravitee-am-gateway-standalone/gravitee-am-gateway-standalone-distribution/src/main/resources/templates/i18n/messages_en.properties [hunk 4]
```diff
@@ -24,3 +31,128 @@ reset_password.email.header.title=Hi {0}, there was a request to reset your pass
...
+mfa_enroll.email=We will send verification code to your email address
```
Comment (farmborough) [inline gravitee-am-gateway/gravitee-am-gateway-standalone/gravitee-am-gateway-standalone-distribution/src/main/resources/templates/i18n/messages_en.properties:101]:
```suggestion
mfa_enroll.email=We will send a verification code to your email address
```

File: gravitee-am-gateway/gravitee-am-gateway-standalone/gravitee-am-gateway-standalone-distribution/src/main/resources/templates/i18n/messages_en.properties [hunk 5]
```diff
@@ -24,3 +31,128 @@ reset_password.email.header.title=Hi {0}, there was a request to reset your pass
...
+mfa_enroll.email.invalid=Invalid email address
+mfa_enroll.http.description=Enter the code provided by the HTTP in next step
```
Comment (farmborough) [inline gravitee-am-gateway/gravitee-am-gateway-standalone/gravitee-am-gateway-standalone-distribution/src/main/resources/templates/i18n/messages_en.properties:103]:
```suggestion
mfa_enroll.http.description=Enter the code provided by the HTTP in the next step
```

File: gravitee-am-gateway/gravitee-am-gateway-standalone/gravitee-am-gateway-standalone-distribution/src/main/resources/templates/i18n/messages_en.properties [hunk 6]
```diff
@@ -24,3 +31,128 @@ reset_password.email.header.title=Hi {0}, there was a request to reset your pass
...
+mfa_enroll.http=Go to the next step and enter the code provided by HTTP
+mfa_enroll.fido.description=Select 'Next' button to trigger the registration process
+mfa_enroll.fido=Select 'Next' button to trigger registration process
+mfa_enroll.on.select.title=Multi-factor Authentication
+mfa_enroll.on.select.description=Configure multi-factor authentication by choosing the method to validate your identity
+
+mfa_challenge.title=Authenticate your account
+mfa_challenge.otp=Please type in the code displayed on your multi-factor authenticator app from your device
+mfa_challenge.sms=Please type in the code sent by SMS on your mobile phone
```
Comment (farmborough) [inline gravitee-am-gateway/gravitee-am-gateway-standalone/gravitee-am-gateway-standalone-distribution/src/main/resources/templates/i18n/messages_en.properties:112]:
```suggestion
mfa_challenge.sms=Please type in the code sent by SMS to your mobile phone
```

File: gravitee-am-gateway/gravitee-am-gateway-standalone/gravitee-am-gateway-standalone-distribution/src/main/resources/templates/i18n/messages_en.properties [hunk 7]
```diff
@@ -24,3 +31,128 @@ reset_password.email.header.title=Hi {0}, there was a request to reset your pass
...
+mfa_challenge.call=You will receive a call shortly. Follow the instructions and type in the given code
+mfa_challenge.email=Please type in the code sent by Email
```
Comment (farmborough) [inline gravitee-am-gateway/gravitee-am-gateway-standalone/gravitee-am-gateway-standalone-distribution/src/main/resources/templates/i18n/messages_en.properties:114]:
```suggestion
mfa_challenge.email=Please type in the code sent by email
```

File: gravitee-am-gateway/gravitee-am-gateway-standalone/gravitee-am-gateway-standalone-distribution/src/main/resources/templates/i18n/messages_en.properties [hunk 8]
```diff
@@ -24,3 +31,128 @@ reset_password.email.header.title=Hi {0}, there was a request to reset your pass
...
+mfa_challenge.fido=Sign in with fingerprint, device or security key
+mfa_challenge.error=Invalid code
+mfa_challenge.button.submit=Verify
+mfa_challenge.remember.device=Remember my device for
+mfa_challenge.alternate=Having trouble ? Try other methods
```
Comment (farmborough) [inline gravitee-am-gateway/gravitee-am-gateway-standalone/gravitee-am-gateway-standalone-distribution/src/main/resources/templates/i18n/messages_en.properties:119]:
```suggestion
mfa_challenge.alternate=Having trouble? Try other methods
```

File: gravitee-am-gateway/gravitee-am-gateway-standalone/gravitee-am-gateway-standalone-distribution/src/main/resources/templates/i18n/messages_en.properties [hunk 9]
```diff
@@ -24,3 +31,128 @@ reset_password.email.header.title=Hi {0}, there was a request to reset your pass
...
+
+mfa_alternative.title=Select a method
+mfa_alternative.description=Try to sign in using other options
+mfa_alternative.otp=Use a verification code displayed on your multi-factor authenticator app from your device
+mfa_alternative.sms=Receive a verification code to your phone to authenticate
+mfa_alternative.email=Send email to your email address
```
Comment (farmborough) [inline gravitee-am-gateway/gravitee-am-gateway-standalone/gravitee-am-gateway-standalone-distribution/src/main/resources/templates/i18n/messages_en.properties:125]:
```suggestion
mfa_alternative.email=Send an email to your email address
```

File: gravitee-am-gateway/gravitee-am-gateway-standalone/gravitee-am-gateway-standalone-distribution/src/main/resources/templates/i18n/messages_en.properties [hunk 10]
```diff
@@ -24,3 +31,128 @@ reset_password.email.header.title=Hi {0}, there was a request to reset your pass
...
+mfa_alternative.http=Enter the code provided by the HTTP in next step
```
Comment (farmborough) [inline gravitee-am-gateway/gravitee-am-gateway-standalone/gravitee-am-gateway-standalone-distribution/src/main/resources/templates/i18n/messages_en.properties:126]:
```suggestion
mfa_alternative.http=Enter the code provided by HTTP in next step
```

File: gravitee-am-gateway/gravitee-am-gateway-standalone/gravitee-am-gateway-standalone-distribution/src/main/resources/templates/i18n/messages_en.properties [hunk 11]
```diff
@@ -24,3 +31,128 @@ reset_password.email.header.title=Hi {0}, there was a request to reset your pass
...
+mfa_alternative.fido=Select 'Next' button to trigger the registration process
```
Comment (farmborough) [inline gravitee-am-gateway/gravitee-am-gateway-standalone/gravitee-am-gateway-standalone-distribution/src/main/resources/templates/i18n/messages_en.properties:127]:
```suggestion
mfa_alternative.fido=Select the 'Next' button to trigger the registration process
```

File: gravitee-am-gateway/gravitee-am-gateway-standalone/gravitee-am-gateway-standalone-distribution/src/main/resources/templates/i18n/messages_en.properties [hunk 12]
```diff
@@ -24,3 +31,128 @@ reset_password.email.header.title=Hi {0}, there was a request to reset your pass
...
+mfa_alternative.recovery.code=Use a recovery code previously generated
+mfa_alternative.submit.button=Next
+
+mfa_recovery.title=Recovery codes
+mfa_recovery.description=Recovery codes are used to access your account when you cannot receive tow-factor authentication codes. Each code can only be used once.
```
Comment (farmborough) [inline gravitee-am-gateway/gravitee-am-gateway-standalone/gravitee-am-gateway-standalone-distribution/src/main/resources/templates/i18n/messages_en.properties:132]:
```suggestion
mfa_recovery.description=Recovery codes are used to access your account when you cannot receive two-factor authentication codes. Each code can only be used once.
```

File: gravitee-am-gateway/gravitee-am-gateway-standalone/gravitee-am-gateway-standalone-distribution/src/main/resources/templates/i18n/messages_en.properties [hunk 13]
```diff
@@ -24,3 +31,128 @@ reset_password.email.header.title=Hi {0}, there was a request to reset your pass
...
+reset_password.title=Set new password
+reset_password.description=The new password must not use your profile information
+reset_password.password.placeholder=Password
+reset_password.confirm.password.placeholder=Confirm password
+reset_password.button.submit=Set new password
+reset_password.success.title=Reset password confirmation
+reset_password.success.description=We have reset your password. Go back to your application to login
+reset_password.error.title=Reset password error
+reset_password.error.description=Please go back to your client application and try again, or contact the owner and ask for support
+reset_password.error.description.label=Error description :
```
Comment (farmborough) [inline gravitee-am-gateway/gravitee-am-gateway-standalone/gravitee-am-gateway-standalone-distribution/src/main/resources/templates/i18n/messages_en.properties:146]:
```suggestion
reset_password.error.description.label=Error description:
```

File: gravitee-am-gateway/gravitee-am-gateway-standalone/gravitee-am-gateway-standalone-distribution/src/main/resources/templates/i18n/messages_fr.properties [hunk 14]
```diff
@@ -1,12 +1,20 @@
 login.page.title=Identifiez-vous
+login.page.description=Vous n'avez pas encore de compte?
 login.page.subtitle=pour continuer vers
 login.page.label.username=Identifiant
 login.page.label.password=Mot de passe
 login.page.error.default.message=Utilisateur inconnu ou invalide
 login.page.button.submit=Connexion
+login.page.forgot.password=Mot de passe oubli� ?
```
Comment (farmborough) [inline gravitee-am-gateway/gravitee-am-gateway-standalone/gravitee-am-gateway-standalone-distribution/src/main/resources/templates/i18n/messages_fr.properties:8]:
```suggestion
login.page.forgot.password=Mot de passe oublié?
```

**Review by farmborough — CHANGES_REQUESTED** [review]
I've taken the opportunity to fix the AM English syntax.

File: gravitee-am-gateway/gravitee-am-gateway-standalone/gravitee-am-gateway-standalone-distribution/src/main/resources/templates/i18n/messages_en.properties [hunk 15]
```diff
@@ -1,12 +1,19 @@
 login.page.title=Sign in
+login.page.description=Don't have an account yet?
 login.page.subtitle=to continue to
 login.page.label.username=Username
 login.page.label.password=Password
 login.page.error.default.message=Wrong user or password
 login.page.button.submit=Sign in
+login.page.forgot.password=Forgot Password ?
```
[the remark this answers is not in the document]
Comment (ashraf706) [reply gravitee-am-gateway/gravitee-am-gateway-standalone/gravitee-am-gateway-standalone-distribution/src/main/resources/templates/i18n/messages_en.properties:8]: done

File: gravitee-am-gateway/gravitee-am-gateway-standalone/gravitee-am-gateway-standalone-distribution/src/main/resources/templates/i18n/messages_en.properties [hunk 16]
```diff
@@ -24,3 +31,128 @@ reset_password.email.header.title=Hi {0}, there was a request to reset your pass
...
+
+forgot_password.title=Change your password
+forgot_password.description=We'll send you reset instructions
+forgot_password.email.placeholder=Email
+forgot_password.button.submit=Send
+forgot_password.success.title=Check your email
+forgot_password.success.description=We have sent a reset password link
+forgot_password.error.title=Forgot password error
+forgot_password.error.description=Please go back to your client application and try again, or contact the owner and ask for support
+forgot_password.error.description.label=Error description :
```
[the remark this answers is not in the document]
Comment (ashraf706) [reply gravitee-am-gateway/gravitee-am-gateway-standalone/gravitee-am-gateway-standalone-distribution/src/main/resources/templates/i18n/messages_en.properties:43]: done

File: gravitee-am-gateway/gravitee-am-gateway-standalone/gravitee-am-gateway-standalone-distribution/src/main/resources/templates/i18n/messages_en.properties [hunk 17]
```diff
@@ -24,3 +31,128 @@ reset_password.email.header.title=Hi {0}, there was a request to reset your pass
...
+registration.description=Already have an account?
+registration.sign.in=Sign in
+registration.button.submit=Register
+registration.first.name.placeholder=First name
+registration.last.name.placeholder=Last name
+registration.user.name=User name
+registration.email.placeholder=Email
+registration.password.placeholder=Password
+registration.success.title=Registered successfully
+registration.success.description=Thank you for creating account
```
[the remark this answers is not in the document]
Comment (ashraf706) [reply gravitee-am-gateway/gravitee-am-gateway-standalone/gravitee-am-gateway-standalone-distribution/src/main/resources/templates/i18n/messages_en.properties:55]: done

File: gravitee-am-gateway/gravitee-am-gateway-standalone/gravitee-am-gateway-standalone-distribution/src/main/resources/templates/i18n/messages_en.properties [hunk 18]
```diff
@@ -24,3 +31,128 @@ reset_password.email.header.title=Hi {0}, there was a request to reset your pass
...
+registration.error.title=Registration failed
+registration.error.description=Please go back to your client application and try again, or contact the owner and ask for support
+registration.error.description.label=Error description :
```
[the remark this answers is not in the document]
Comment (ashraf706) [reply gravitee-am-gateway/gravitee-am-gateway-standalone/gravitee-am-gateway-standalone-distribution/src/main/resources/templates/i18n/messages_en.properties:58]: done

File: gravitee-am-gateway/gravitee-am-gateway-standalone/gravitee-am-gateway-standalone-distribution/src/main/resources/templates/i18n/messages_en.properties [hunk 19]
```diff
@@ -24,3 +31,128 @@ reset_password.email.header.title=Hi {0}, there was a request to reset your pass
...
+password.validation.label=Password must contains:
+password.minLength.before=Contains at least
+password.minLength.after=characters
+password.include.numbers=Contains a number
+password.include.special.characters=Contains a special character
+password.letters.mixed.cases=Contains letters in mixed case
+password.max.consecutive.letters.before=Max
+password.max.consecutive.letters.after=Consecutive letters or numbers
+password.exclude.common.passwords=Don't use common names or passwords
+password.exclude.user.info=Don't use your profile information in password
```
[the remark this answers is not in the document]
Comment (ashraf706) [reply gravitee-am-gateway/gravitee-am-gateway-standalone/gravitee-am-gateway-standalone-distribution/src/main/resources/templates/i18n/messages_en.properties:69]: done

File: gravitee-am-gateway/gravitee-am-gateway-standalone/gravitee-am-gateway-standalone-distribution/src/main/resources/templates/i18n/messages_en.properties (hunk 10 above)
Comment (ashraf706) [reply gravitee-am-gateway/gravitee-am-gateway-standalone/gravitee-am-gateway-standalone-distribution/src/main/resources/templates/i18n/messages_en.properties:126]: done

File: gravitee-am-gateway/gravitee-am-gateway-standalone/gravitee-am-gateway-standalone-distribution/src/main/resources/templates/i18n/messages_en.properties (hunk 11 above)
Comment (ashraf706) [reply gravitee-am-gateway/gravitee-am-gateway-standalone/gravitee-am-gateway-standalone-distribution/src/main/resources/templates/i18n/messages_en.properties:127]: done

File: gravitee-am-gateway/gravitee-am-gateway-standalone/gravitee-am-gateway-standalone-distribution/src/main/resources/templates/i18n/messages_en.properties (hunk 12 above)
Comment (ashraf706) [reply gravitee-am-gateway/gravitee-am-gateway-standalone/gravitee-am-gateway-standalone-distribution/src/main/resources/templates/i18n/messages_en.properties:132]: done

File: gravitee-am-gateway/gravitee-am-gateway-standalone/gravitee-am-gateway-standalone-distribution/src/main/resources/templates/i18n/messages_en.properties (hunk 13 above)
Comment (ashraf706) [reply gravitee-am-gateway/gravitee-am-gateway-standalone/gravitee-am-gateway-standalone-distribution/src/main/resources/templates/i18n/messages_en.properties:146]: done

File: gravitee-am-gateway/gravitee-am-gateway-standalone/gravitee-am-gateway-standalone-distribution/src/main/resources/templates/i18n/messages_fr.properties (hunk 14 above)
Comment (ashraf706) [reply gravitee-am-gateway/gravitee-am-gateway-standalone/gravitee-am-gateway-standalone-distribution/src/main/resources/templates/i18n/messages_fr.properties:8]: done

File: gravitee-am-gateway/gravitee-am-gateway-standalone/gravitee-am-gateway-standalone-distribution/src/main/resources/templates/i18n/messages_fr.properties [hunk 20]
```diff
@@ -23,3 +31,128 @@ reset_password.email.header.title=Bonjour {0}, une demande de r
...
+
+webauthn.login.description=Utilisation de l'empreinte digitale, de l'appareil ou de la cl� de s�curit�
+webauthn.login.button.next=Suivant
+webauthn.login.tips=Suivez les instructions dans la fen�tre contextuelle de s�curit�
+webauthn.login.error=�chec de la connexion
+webauthn.login.error.description=Utilisateur invalide
+
+mfa.otp=Mot de passe � usage unique
+mfa.sms=SMS
+mfa.email=Courriel
```
Comment (leleueri) [inline gravitee-am-gateway/gravitee-am-gateway-standalone/gravitee-am-gateway-standalone-distribution/src/main/resources/templates/i18n/messages_fr.properties:85]:
```suggestion
mfa.email=Courriel
```
I think it is ok to keep "email"

File: gravitee-am-gateway/gravitee-am-gateway-standalone/gravitee-am-gateway-standalone-distribution/src/main/resources/templates/i18n/messages_fr.properties [hunk 21]
```diff
@@ -23,3 +31,128 @@ reset_password.email.header.title=Bonjour {0}, une demande de r
...
+
+reset_password.title=D�finir un nouveau mot de passe
+reset_password.description=Le nouveau mot de passe ne doit pas utiliser les informations de votre profil
+reset_password.password.placeholder=Mot de passe
+reset_password.confirm.password.placeholder=Confirmer le mot de passe
+reset_password.button.submit=D�finir un nouveau mot de passe
+reset_password.success.title=R�initialiser la confirmation du mot de passe
+reset_password.success.description=Nous avons r�initialis� votre mot de passe. Retournez � votre application pour vous connecter
+reset_password.error.title=Erreur de r�initialisation du mot de passe
+reset_password.error.description=Veuillez revenir � votre application client et r�essayer, ou contacter le propri�taire et demander de l'aide
```
Comment (leleueri) [inline gravitee-am-gateway/gravitee-am-gateway-standalone/gravitee-am-gateway-standalone-distribution/src/main/resources/templates/i18n/messages_fr.properties:145]:
```suggestion
reset_password.error.description=Veuillez revenir à votre application client et réessayer, ou contacter votre administrateur
```

**Review by leleueri — CHANGES_REQUESTED** [review]
In the WebAuthn login page, a span isn't translated :
image


**NOTE:** These translation need to be available into the manage api too (for preview mode)

**Comment by ashraf706** [comment]
> In the WebAuthn login page, a span isn't translated : image
> 
> **NOTE:** These translation need to be available into the manage api too (for preview mode)

Updated the description with property

**Review by leleueri — CHANGES_REQUESTED** [review]
Two things:
* The form "registration_confirmation.html" has no entries into the en/fr properties files. (Change required)
* naming in the key are not consistent. for example, the `login` page keys start by "`login.page`" and the emails like `registration_confirmation` has keys that start by "`registration_confirmation.email`". For all new keys, `page` is missing from the key. Maybe it a convention we should put in place to make the distinction between keys that target a form and the one that target an email. I also wondering if the first element of the key should be the template name followed by the type (page or email) for example the mfa_challenge is a template name for both page and email (ex:  mfa_challenge.page.title / mfa_challenge.email.header.title) :thinking: wdyt ?

**Comment by ashraf706** [comment]
> 



> Two things:
> 
> * The form "registration_confirmation.html" has no entries into the en/fr properties files. (Change required)

**Done**

> * naming in the key are not consistent. for example, the `login` page keys start by "`login.page`" and the emails like `registration_confirmation` has keys that start by "`registration_confirmation.email`". For all new keys, `page` is missing from the key. Maybe it a convention we should put in place to make the distinction between keys that target a form and the one that target an email. I also wondering if the first element of the key should be the template name followed by the type (page or email) for example the mfa_challenge is a template name for both page and email (ex:  mfa_challenge.page.title / mfa_challenge.email.header.title) 🤔 wdyt ?

**Review by leleueri — APPROVED** [review]

**Pull request merged**